src/driver/ExpressDriver.ts

```diff
@@ -175,7 +175,7 @@ export class ExpressDriver extends BaseDriver implements Driver {
                     return request.session;
                 }
             case ParamTypes.STATE:
-                throw new Error("@State decorators are not supported by ExpressDriver yet.");
+                return undefined;
```

Comment:
Now you removed the good line 😉 
`throw new Error("@State decorators are not supported by ExpressDriver yet.");` should stay to tell user that it won't work with Express